test/index.test.js

```diff
@@ -298,17 +299,12 @@ describe('test `retryInit` function', () => {
     });
 });
 
-describe('test fetch retry', () => {
-    afterEach(() => {
-        assert(nock.isDone);
-        nock.cleanAll();
-    });
-
+async function testFetchRetry(options = {}){
     it('test fetch get works 200', async () => {
         nock(FAKE_BASE_URL)
             .get(FAKE_PATH)
             .reply(200, { ok: true });
-        const response = await fetch(`${FAKE_BASE_URL}${FAKE_PATH}`, { method: 'GET' });
+        const response = await fetch(`${FAKE_BASE_URL}${FAKE_PATH}`, { method: 'GET', ...options });
```

Comment:
super nitpick test case i just thought of, add a test without passing `,...options` just to make sure it works as before. (I know we default options to `{}`: https://github.com/adobe/node-fetch-retry/blob/930a3e61077398a557ec129070dc545f4cb5ff22/index.js#L200, so this test would serve as a safeguard in case it gets removed)